templates/rails/files/.platform/routes.yaml

```diff
@@ -0,0 +1,3 @@
+"https://{default}":
```

Comment:
Please, don't forget to put details on this file, e.g.:
https://github.com/platformsh/template-drupal8/blob/master/.platform/routes.yaml